src/openassetio-core/include/openassetio/Context.hpp

```diff
@@ -129,6 +129,16 @@ class Context final {
    */
   managerApi::ManagerStateBasePtr managerState{};
```

Comment:
[consider][minor] Here and for `locale` it would perhaps be clearer to a reader if we get rid of the default initialisation (i.e. the `{}`).

Reply:
Removed [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/495/commits/9e028fe534c2d59bcc6dfd2ca1caa704ed2afce6).